ironfish-rust/src/merkle_note.rs

```diff
@@ -48,7 +50,7 @@ pub struct MerkleNote {
     pub(crate) note_commitment: Scalar,
 
     /// Public part of ephemeral diffie-hellman key-pair. See the discussion on
-    /// keys::shared_secret to understand how this is used
+    /// [`shared_secret`] to understand how this is used
```

Comment:
We can link in docs! This lets us A) go-to-definition in editors and B) would create links in the auto-generated rust docs, if/when we start using that.